client/src/js/app.js

```diff
@@ -979,15 +983,18 @@ function localeConfig(tmhDynamicLocaleProvider) {
 }
 
 // redirect to login if not signed in.
-function startupConfig($rootScope, $state, SessionService, amMoment, Notify) {
+function startupConfig($rootScope, $state, SessionService, amMoment, Notify, $location) {
 
   // make sure the user is logged in and allowed to access states when
   // navigating by URL.  This is pure an authentication issue.
   $rootScope.$on('$locationChangeStart', function (event, next) {
-
     var isLoggedIn = !!SessionService.user;
     var isLoginState = next.indexOf('#/login') !== -1;
 
+    if($location.$$path === '/error403'){
```

Comment:
Using (next.indexOf('/error404')  if we modify the path by putting each time an unauthorized path after 3 tentantive we reach accessed  at even found in the address / bar error403, 
That's why I'll use
if (next.indexOf('/error403') !== -1) {
  $state.go('/error403');
}